apps/website/src/pages/docs/avatar.mdx

```diff
@@ -17,81 +17,59 @@ import * as AllExamples from 'examples';
+- `abbreviation` prop allows users to provide a short abbreviation or initials for the avatar.
+- `image` prop can be passed to set a custom image source or icon that will be displayed within the avatar.
 
-<LiveExample src='Avatar.initials.tsx'>
-  <AllExamples.AvatarInitialsExample client:load />
+<LiveExample src='Avatar.types.tsx'>
+  <AllExamples.AvatarTypesExample client:load />
 </LiveExample>
 
-### Icon
+It is recommended to display an avatar with initials or an icon for users without a profile picture. It should show one or two letters or the first letter of their email if no name is provided.
 
-An icon may be display instead of initials, typically used by organizations instead of a person.
-
-<LiveExample src='Avatar.icon.tsx'>
-  <AllExamples.AvatarIconExample client:load />
-</LiveExample>
-
-### Picture
+### Sizes
 
-If a user uploads a photo we display their photo instead of their initials.
+Avatars can be displayed in various sizes including `small`, `medium`, `large`, and `x-large`.
 
-<LiveExample src='Avatar.picture.tsx'>
-  <AllExamples.AvatarPictureExample client:load />
+<LiveExample src='Avatar.sizes.tsx'>
+  <AllExamples.AvatarSizesExample client:load />
 </LiveExample>
 
 ### Status indicators
 
-You may want to indicate the online status of a user. Status indicators display in the bottom right of the avatar. We suggest not using statuses on the small size avatar. The available statuses are:
+Status indicators are indicators that display in the bottom right of the avatar. Statuses can be set using the `status` prop. The available statuses are:
 
 - `online`
 - `away`
 - `busy`
 - `offline`
 
+We suggest not using statuses on the small size avatar.
+
 <LiveExample src='Avatar.statuses.tsx'>
   <AllExamples.AvatarStatusesExample client:load />
 </LiveExample>
 
-## Avatar group
+## Avatar Group
 
-Avatars can be displayed in a list to represent a group of people. This visual treatment reinforces the fact that these people work together and/or share similar permissions to access/view/edit a file or a project. For example, you might display the list of people who previously edited an item, are all collaborating on a specific activity, etc. To save space, consider the "Stacked" option below.
-
-<LiveExample src='Avatar.group.tsx'>
-  <AllExamples.AvatarGroupExample client:load />
-</LiveExample>
+Avatars can be displayed in a list to represent a group of people using `<AvatarGroup>` component.
 
 ### Stacked
 
-To save on space, the avatars can overlay each other. Note that when using the stacked group that you can not display [status indicators](#status-indicators).
+By default or passing `stacked` prop, the avatars can overlay each other to save space. Note that when using the stacked group that you can not display [status indicators](#status-indicators).
```

Comment:
Can be rephrased slightly to clearly describe default vs non-default behavior

```suggestion
By default, the avatars will overlay each other to save space. This can be turned off by setting `stacked={false}` if needed.

Note that when using the stacked group, you can not display [status indicators](#status-indicators).
```